modules/hrm/views/announcements.php

```diff
@@ -0,0 +1,42 @@
+<div class="wrap erp erp-hr-announcement">
+    <h2>
+        <?php esc_html_e( 'People', 'erp' ); ?>
+        <a href="<?php echo admin_url( 'post-new.php?post_type=erp_hr_announcement' ); ?>" id="erp-new-announcement" data-single="1" class="add-new-h2"><?php esc_html_e( 'Add New', 'erp' ); ?></a>
+    </h2>
+
+    <?php if ( ! empty( $_GET['announcement_delete'] ) ) { ?>
+        <div id="message" class="error notice is-dismissible below-h2">
+            <p><?php esc_html_e( 'Some announcements doesn\'t delete', 'erp' ); ?></p>
```

Comment:
Change the sentence. You might use `esc_html_e( 'Some announcements could not be deleted', 'erp' );`